chpl/chpl-api/e2e/collections/surveillance-controller.postman_collection.json

```diff
@@ -0,0 +1,382 @@
+{
+	"info": {
+		"name": "surveillance-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "POST /surveillance/upload - access denied to anonymous user",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"surveillance upload request by anonymous user should return Status code 401\", function () {\r",
+							"    pm.response.to.have.status(401);\r",
+							"});\r",
+							"\r",
+							"pm.test(\"anonymous user should be denied access to upload surveillance\", function () {\r",
+							"    pm.expect(pm.response.text()).to.include(\"Access is denied\");\r",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "API-KEY",
+						"value": "{{apiKey}}",
+						"type": "text"
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "file",
+							"type": "file",
+							"src": "chpl/chpl-api/e2e/data/SurveillanceUpload.csv"
+						}
+					]
+				},
+				"url": {
+					"raw": "{{url}}/rest/surveillance/upload",
+					"host": [
+						"{{url}}"
+					],
+					"path": [
+					    "rest",
+						"surveillance",
+						"upload"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "POST /surveillance/upload - ROLE_ACB has access to upload surveillance activity on own listings",
```

Comment:
Can we also add a test mentioned in TC document as "ACB A attempts to add surveillance activity on listing owned ACB B via upload"?
This test is not in UI test so good to add it here.

Reply:
I'd like to add that test too but I put that one in excluded list due to lack of aqa user account that is attached to one ACB organization only. The one we have now is attached to all active ACB organizations, it won't work for this scenario.

Reply:
I see. We will need to request new ACB credentials from Andrew. so we can start having that user for these types of test. Not holding PR for this.